tree/20_cfe_basics/log_rudder.cf

```diff
@@ -728,3 +733,27 @@ bundle agent endExecution
 * Any existing configuration policy will continue to be applied without change. *
 *********************************************************************************";
 }
+bundle agent rudder_set_force_component_none(mode)
```

Comment:
you should advertise that it is necessary to call it twice: once to set the mode, and once to remove the mode, otherwise all reports would be Nonde

Reply:
also, that calling it with any text but "false" would make futur reports None